### DISCUSSION TITLE
mapStyle/Basemap not showing

### DISCUSSION THREAD

**Discussion by WilliamScott-AdventGX** [visgl/deck.gl#7298] [Q&A] [answered]
With AngularTS, I copied the JS from the demo on this webpage: https://deck.gl/docs/api-reference/core/map-view

The demo imports did not work, so I changed them to
```
import { Deck } from '@deck.gl/core';
import { ScatterplotLayer } from '@deck.gl/layers';
import { MapView } from '@deck.gl/core';
import Map from '@arcgis/core/Map';
```

Now, the scatterplot loads but the background remains white.

Example photo: 
<img width="1060" alt="Screen Shot 2022-09-27 at 10 45 57 AM" src="https://user-images.githubusercontent.com/109538533/192573383-f967c330-33fd-495f-a7ae-3d680de71c5c.png">

Here is the code copied from the example:

```
const map = new Map({
      basemap: 'dark-gray-vector',
      layers: []
    });

    new Deck({
      views: new MapView({
        container: 'root',
        repeat: true,
        // nearZMultiplier: 0.1,
        // farZMultiplier: 1.01,
        // orthographic: true,
        map: map,
      }),
      initialViewState: {
        longitude: 0,
        latitude: 10,
        zoom: 1,
        pitch: 0,
        bearing: 0,
        minZoom: 0,
        maxZoom: 20,
        minPitch: 0,
        maxPitch: 60
      },
      controller: true,
      mapStyle: 'https://basemaps.cartocdn.com/gl/positron-nolabels-gl-style/style.json',
      layers: [
        new ScatterplotLayer({
          data: 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/website/airports.json',
          getPosition: d => d.coordinates,
          getRadius: 100,
          getColor: [155, 40, 0],
          radiusMinPixels: 2
        })
      ],
      container: 'root',
    });
```

Any ideas on why the mapStyle/basemap is not loading? Anything helps, thanks!

**Comment by Pessimistress** [comment, marked as answer]
The codepen on the documentation page uses mapbox-gl. To use arcgis please refer to https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/arcgis

**Reply by WilliamScott-AdventGX** [reply]
This got me going in the right direction. This is the right answer however I wanted to stay with Mapbox. I was able to use an [ArcLayer via a DeckOverlay on a mapbox map](https://github.com/visgl/deck.gl/blob/master/examples/get-started/pure-js/mapbox/app.js).

I appreciate the answer.
